modules/statics/src/coins.ts

```diff
@@ -158,6 +162,7 @@ const ALGO_FEATURES = [
   CoinFeature.MULTISIG_COLD,
   CoinFeature.BULK_TRANSACTION,
 ];
+const THTETH_TOKEN_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.BULK_TRANSACTION];
```

Comment:
If this is intended for HTETH tokens, then note that HTETH itself is a testnet network, so we can keep it as below
```suggestion
const HTETH_TOKEN_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.BULK_TRANSACTION];
```

Reply:
+1